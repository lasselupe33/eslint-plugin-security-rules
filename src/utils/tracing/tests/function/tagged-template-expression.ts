function myTag(strings: TemplateStringsArray, ...expressions: unknown[]) {
  return `${strings[0]}-${expressions[0]}-tagged`;
}

const start = myTag`hello-${2}`;

export {};
