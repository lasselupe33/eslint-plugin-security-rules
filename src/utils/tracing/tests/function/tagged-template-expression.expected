hello--2-tagged